src/components/tap-click/ripple.ts

```diff
@@ -14,92 +14,95 @@ export class RippleActivator extends Activator {
   }
 
   downAction(ev: UIEvent, activatableEle: HTMLElement, startCoord: Coordinates) {
-    let self = this;
-    if (self.disableActivated(ev)) {
+    if (this.disableActivated(ev)) {
       return;
     }
 
     // queue to have this element activated
-    self._queue.push(activatableEle);
-
-    nativeRaf(function() {
```

Comment:
@adamdbradley before merging! I accidentally removed `nativeRaf`. but, what was it there?

I would like to document it, so it is not removed again the future...